docs/en/status-communities/delegate-admin-functions-to-your-community.md

```diff
@@ -0,0 +1,60 @@
+---
+id: 543
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Delegate admin functions to your community
+
+!!! note ""
+    Currently, you can only delegate admin functions using Status desktop. [:octicons-desktop-download-16: Download Status desktop][status-web-download].
+
+Community owners have total control over their communities, including settings, membership approvals and permissions. In addition, owners can delegate administrative functions to other community members through the [admin role][about-the-owner-and-admin-community-roles].
+
+To delegate admin functions to others, owners use [token-based community permissions][set-up-your-community-permissions]. Tokens are digital assets in a blockchain (such as the [Ethereum blockchain][about-the-ethereum-blockchain]), including assets (virtual coins), collectibles and ENS names.
+
+## What to expect
+
+- To delegate admin functions, you assign the `Become admin` community permission to a collectible or ENS name owner.
+- Using ENS names to delegate admin functions is more straightforward than using collectibles, but it requires the admin to own the ENS name.
+- You can create a maximum of five different `Become admin` community-level permissions, but you can apply a single permission to multiple members.
+- To stop delegating admin functions, [self-destruct the admin token][self-destruct-a-token-remotely] or [delete the admin permission][delete-community-permissions].
+
+## How to delegate admin functions
+
+You can delegate [admin functions][about-the-owner-and-admin-community-roles] to others using a collectible or an ENS name.
+
+If you use a collectible, follow this process:
+
+- [x] [Mint][mint-tokens-for-your-community] a non-transferable (soulbound) and remotely-destructible collectible.
+- [x] [Airdrop the collectible][how-to-airdrop-collectibles-in-status] to the person you wish to delegate admin functions to.
+- [x] [Delegate the admin function](#delegate-admin-functions-using-collectibles-or-ens-names) using token-based community permissions.
+
+Using ENS names, you don't need to mint and airdrop any token. You [delegate admin functions](#delegate-admin-functions-using-collectibles-or-ens-names) to a specific ENS name owner (such as alice.acme.eth) or a group of people under the same ENS domain (such as acme.eth).
+
+!!! note
+    Assets don't have the non-transferable (soulbound) and self-destructing properties collectibles have, making it harder to remove admin functions from other users. Avoid using assets to delegate admin functions.
+
+## Delegate admin functions using collectibles or ENS names
+
+=== "Desktop"
+
+    1. From the navigation sidebar, click your community.
+    1. On top of the channel sidebar, click your community logo and then, click **Permissions**.
+    1. Click **Add permission**.
+    1. For the `Who holds` variable, choose your delegation option and use the table below for reference.
```

Comment:
Can link the H2 title of the above section to “choose your delegation option”

Reply:
Great idea 👍